front/app/containers/Admin/projects/project/analysis/FilterItems/index.tsx

```diff
@@ -1,138 +1,140 @@
-    domicileField?.id
-  );
-
-  const genderUrlQueryParamKey = `author_custom_${genderField?.id}`;
-  const domicileUrlQueryParamKey = `author_custom_${domicileField?.id}`;
-  const birthyearUrlQueryParamFromKey = `author_custom_${birthyearField?.id}_from`;
-  const birthyearUrlQueryParamToKey = `author_custom_${birthyearField?.id}_to`;
-
-  const translationKeys: Record<string, string> = {
-    search: 'Search: ',
-    published_at_from: 'Start: ',
-    published_at_to: 'End: ',
-    reactions_from: 'Reactions > ',
-    reactions_to: 'Reactions < ',
-    votes_from: 'Votes > ',
-    votes_to: 'Votes < ',
-    comments_from: 'Comments > ',
-    comments_to: 'Comments < ',
-    [genderUrlQueryParamKey]: 'Gender: ',
-    [domicileUrlQueryParamKey]: 'Domicile: ',
-    [birthyearUrlQueryParamFromKey]: 'Birthyear > ',
-    [birthyearUrlQueryParamToKey]: 'Birthyear < ',
+  const translationKeys: Record<
+    string,
+    { translationKey: string; predicate: '>' | '<' | '=' }
+  > = {
+    search: {
+      translationKey: 'Search',
+      predicate: '>',
+    },
+    published_at_from: {
+      translationKey: 'Start',
+      predicate: '>',
+    },
+    published_at_to: {
+      translationKey: 'End',
+      predicate: '<',
+    },
+    reactions_from: {
+      translationKey: 'Reactions',
+      predicate: '>',
+    },
+    reactions_to: {
+      translationKey: 'Reactions',
+      predicate: '<',
+    },
+    votes_from: {
+      translationKey: 'Votes',
+      predicate: '>',
+    },
+    votes_to: {
+      translationKey: 'Votes',
+      predicate: '<',
+    },
+    comments_from: {
+      translationKey: 'Comments',
+      predicate: '>',
+    },
+    comments_to: {
+      translationKey: 'Comments',
+      predicate: '<',
+    },
   };
 
-  const filterEntries = Object.entries(filters)
-    .filter(([key]) => key !== 'tag_ids')
-    .filter(([key]) => !(isEditable && key === 'search'));
-
-  const filterItemDisplayValue = (
-    key: string,
-    value: string | string[] | undefined | number
-  ) => {
-    switch (key) {
-      case genderUrlQueryParamKey:
-        return (
-          isArray(value) &&
-          value.map((value: string) => (
-            <Box key={value} ml="4px">
-              {localize(
-                genderOptions?.data.find((option) => {
-                  return option.attributes.key === value;
-                })?.attributes.title_multiloc
-              )}
-            </Box>
-          ))
-        );
-      case domicileUrlQueryParamKey:
-        return (
-          isArray(value) &&
-          value.map((value: string) => (
-            <Box key={value} ml="4px">
-              {localize(
-                domicileOptions?.data.find(
-                  (option) => option.attributes.key === value
-                )?.attributes.title_multiloc
-              )}
-            </Box>
-          ))
-        );
-
-      default:
-        return value;
-    }
-  };
+  const filterEntries = Object.entries(filters).filter(
```

Comment:
The search should still be filtered out here when it's editable (meaning the component is displayed on the top bar) because we already have the search box there.

Reply:
Ah yes, you're right. I think I accidently deleted this while merging.